mscore/musescore.cpp

```diff
@@ -6876,9 +6888,13 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
             qDebug("2:unknown cmd <%s>", qPrintable(cmd));
         }
     }
+    // update debugger and score tree debugger
     if (debugger) {
         debugger->reloadClicked();
     }
+    if (scoreTreeWidget) {
+        scoreTreeWidget->setScore(cs);
+    }
```

Comment:
Remove this. It should not be necessary to set the score after every command. Doing it in `setCurrentScoreView()` should be enough.

Reply:
Actually I put this here to reload the ScoreItemModel in the Tree view every time a re-layout happens. Otherwise it just crashes when we try to add a second voice or staff, for example. Is there any other place I could put these lines perhaps? Or some other way to reload the model?

Reply:
I see. In that case this [StackOverflow answer](https://stackoverflow.com/a/21487724) might help, or it might just create more trouble than it's worth. For now just leave a comment to say why you did this and we'll come back to it later, probably in another PR.